lib/inputs/rhel/static.ts

```diff
@@ -0,0 +1,27 @@
+import { ExtractAction, ExtractedLayers } from "../../extractor/types";
+import { streamToJson } from "../../stream-utils";
+
+export const getRedHatReposContentAction: ExtractAction = {
+  actionName: "redhat-content-manifests",
+  filePathMatches: (filePath) => isRedHatContentManifest(filePath),
```

Comment:
Is there a reason for the isRedHatContentManifest to be wrapped in another function here, instead of using `filePathMatches: isRedHatContentManifest,`? :)

Reply:
No reason 🙈 Missed it, thanks!